ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/YesNoMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+
+@Component
+public class YesNoMapper implements Mapper<CCDYesNoOption, YesNoOption> {
```

Comment:
Unit tests for YesNoMapper if you need this mapper. I feel this is unnecessary to have mapper for basic types.

Reply:
Already there you missed probably.